import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...) or get_code_authorization_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.OutboundApi();
body = PureCloudPlatformClientV2.DialerAuditRequest() # DialerAuditRequest | AuditSearch
page_size = 25 # int | Page size (optional) (default to 25)
page_number = 1 # int | Page number (optional) (default to 1)
sort_by = ''entity.name'' # str | Sort by (optional) (default to 'entity.name')
sort_order = ''ascending'' # str | Sort order (optional) (default to 'ascending')
facets_only = False # bool | Facets only (optional) (default to False)

try:
    # Retrieves audits for dialer.
    api_response = api_instance.post_outbound_audits(body, page_size=page_size, page_number=page_number, sort_by=sort_by, sort_order=sort_order, facets_only=facets_only)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling OutboundApi->post_outbound_audits: %s\n" % e)